dev/com.ibm.ws.jpa.tests.spec10.entity_fat.common/fat/src/com/ibm/ws/jpa/tests/spec10/entity/tests/Entity_EJB.java

```diff
@@ -244,11 +244,18 @@ public boolean include(ArchivePath arg0) {
             ConfigElementList<ClassloaderElement> cel = appRecord.getClassloaders();
             ClassloaderElement loader = new ClassloaderElement();
             loader.getCommonLibraryRefs().add("HibernateLib");
+            loader.getCommonLibraryRefs().add("AnonymousJDBCLib");
```

Comment:
This appears to be a requirement for old versions of OpenJPA (2.2.x because it was fixed in 3.0.0) and for Oracle only. I would rather not add the JDBC Library to all providers classpaths for all databases. It might mask real issues that should be investigated

Reply:
How about the following instead?

```java
        /*
         * TODO: OpenJPA 2.2.x (JPA 2.0) has hard dependencies on the Oracle JDBC driver classes and
         * therefore requires the driver be added to the application classloader
         *
         * https://issues.apache.org/jira/projects/OPENJPA/issues/OPENJPA-2602
         * https://issues.apache.org/jira/projects/OPENJPA/issues/OPENJPA-2690
         */
        if (AbstractFATSuite.repeatPhase != null && AbstractFATSuite.repeatPhase.contains("20")
            && DatabaseVendor.ORACLE.equals(getDbVendor())) {
            ConfigElementList<ClassloaderElement> cel = appRecord.getClassloaders();
            ClassloaderElement loader = new ClassloaderElement();
            loader.getCommonLibraryRefs().add("AnonymousJDBCLib");
            cel.add(loader);
        }
```

Reply:
Done.